config/initializers/paperclip.rb

```diff
@@ -1,3 +1,7 @@
+Paperclip.interpolates :rails_relative_url_root  do |attachment, style|
```

Comment:
Unused block argument - `attachment`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `style`. You can omit all the arguments if you don't care about them.<br>Unnecessary spacing detected.